st2common/st2common/rbac/types.py

```diff
@@ -50,7 +50,7 @@ class PermissionType(Enum):
 
     # Note: Execution create is granted with "action_execute"
     EXECUTION_VIEW = 'execution_view'
-    EXECUTION_RE_RUN = 'execution_re_run'
+    EXECUTION_RE_RUN = 'execution_rerun'
```

Comment:
See the implementation of `get_resource_type`

`get_resource_type('execution_re_run') == execution_re` and what we want is get_resource_type('excution_rerun') == execution`

`_` can eutehr be in type name or permission name.